tests/dummy/app/templates/nodes/detail/index.hbs

```diff
@@ -151,14 +102,11 @@
     <h2> Create multiple components (bulk) </h2>
     <div>
         <h3> First Component </h3>
-        <label>Title:</label>
-        {{input type="text" value=child1 placeholder="Component's title" required=true}}
+        <label>Title:</label> {{input type="text" value=child1 placeholder="Component's title" required=true}}
         <h3> Second Component </h3>
-        <label>Title:</label>
-        {{input type="text" value=child2 placeholder="Component's title" required=true}}
+        <label>Title:</label> {{input type="text" value=child2 placeholder="Component's title" required=true}}
         <p><button {{action "addChildren" child1 child2 }}>Create</button></p>
```

Comment:
Good call. I'm feeling that we should strongly discourage bulk creates, and will remove this piece of UI.